src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              className={cn({
+                'is-active': (selectedTab.id === tab.id),
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                onClick={() => {
+                  if (tab.id !== selectedTab.id) {
+                    onTabSelected(tab);
+                  }
+                }}
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {(tabs.find(tab => tab.id === selectedTab.id)).content}
```

Comment:
please don't call a function from the template because it will be executed every time when the state change, so make sense to move it to the variable